transport/http/outbound.go

```diff
@@ -41,61 +45,61 @@ var (
 	errOutboundNotStarted     = errors.ErrOutboundNotStarted("http.Outbound")
 )
 
-type outboundConfig struct {
-	keepAlive time.Duration
-}
+// NewOutbound is deprecated
+// TODO rename this func NewOutboundFromURL
+func NewOutbound(urlStr string) transport.Outbound {
```

Comment:
actually since this function now uses an agent under the hood we can switch out the OutboundOption with AgentOption and it should be compatible (as long as they haven't created any custom OutboundOptions, but that would probably be rare.